src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php

```diff
@@ -50,6 +50,11 @@ class SymfonyTestsListenerTrait
      */
     public function __construct(array $mockedNamespaces = [])
     {
+        if ('0' !== getenv('SYMFONY_PHPUNIT_LOCALE')) {
+            // Enforce a consistent locale
+            setlocale(\LC_ALL, 'C');
```

Comment:
Not sure about the `SYMFONY_PHPUNIT_LOCALE` if we want to do:
```
$locale = getenv('SYMFONY_PHPUNIT_LOCALE') :? 'C';
setlocale(\LC_ALL, $locale);
```

or just allow the SYMFONY_PHPUNIT_LOCALE env variable to avoid the set locale call